postgres.go

```diff
@@ -308,6 +308,8 @@ func buildConnInfo(db DBConf) string {
 		certsRequired = true
 	case "verify-peer":
 		certsRequired = true
+	case "verify-full":
+		certsRequired = true
```

Comment:
This should no be an addition, it should replace ` verify-peer`

Reply:
Right, it is done now